mmds/src/lib.rs

```diff
@@ -10,10 +9,10 @@ extern crate micro_http;
 
 pub mod data_store;
 
-use std::sync::{Arc, Mutex};
-
 use data_store::{Error as MmdsError, Mmds};
 use micro_http::{Body, Request, RequestError, Response, StatusCode, Version};
+use serde_json::{Map, Value};
```

Comment:
style nit: we separate the import's related to external crate from the local imports.
So, I would suggest:
```
use serde_json::{Map, Value};
use std::sync::{Arc, Mutex};

use data_store::{Error as MmdsError, Mmds};
use micro_http::{Body, Request, RequestError, Response, StatusCode, Version};
```